decentraland/unity-explorer
# fix: authentication urls & dApp avatar preview

Sets the authentication urls to `org` which is currently prod. Allows to set a secondary development url for development builds.

# Test steps

1. Open the app and follow the authentication process
2. When redirected to the web browser to sign the request observe that your avatar has your wearables
3. Continue through the authentication process, everything should keep working as expected
